Sources/SnowManLogicSpecs/guessManagerSpec.swift

```diff
@@ -98,67 +98,58 @@ class GuessManagerSpec: Swiftest.Spec {
                 	}
 
-                	it("should not add guess to incorrectGuesses collection if it is in the word") {
-                    	let guess = "a"
-                    		guessManager.appendGuess(word, guess: guess)
-                    		expect(guessManager.incorrectGuesses).notTo.contain(guess)
+                	it("should not add word to incorrectGuesses if it is a correctly guessed word") {
+                        	let guess = "apple"
+                        	guessManager.appendGuess(word, guess: guess)
+                        	expect(guessManager.incorrectGuesses).notTo.contain(guess)
                 	}
 
-                	it("should add guess to incorrectGuesses collection if it is not in the word") {
-                    	let guess = "b"
-                    		guessManager.appendGuess(word, guess: guess)
-                    		expect(guessManager.incorrectGuesses).to.contain(guess)
+                	it("should add word to incorrectGuesses if it is an incorrectly guessed word") {
+                        	let guess = "banana"
+                        	guessManager.appendGuess(word, guess: guess)
+                        	expect(guessManager.incorrectGuesses).to.contain(guess)
+                	}
+
+                	it("should not add word to correctGuesses if it is an incorrectly guessed word") {
+                        	let guess = "banana"
+                        	guessManager.appendGuess(word, guess: guess)
+                        	expect(guessManager.correctGuesses).notTo.contain(guess)
+                	}
+                
+             		it("should add guess to correctGuesses collection if it is in the word") {
+                    		let guess = "a"
+               			guessManager.appendGuess(word, guess: guess)
+               			expect(guessManager.correctGuesses).to.contain(guess)
+               		}
+
+               		it("should not add guess to incorrectGuesses collection if it is in the word") {
+                    		let guess = "a"
+               			guessManager.appendGuess(word, guess: guess)
+               			expect(guessManager.incorrectGuesses).notTo.contain(guess)
+               		}
+
+               		it("should add guess to incorrectGuesses collection if it is not in the word") {
+                    		let guess = "b"
+                		guessManager.appendGuess(word, guess: guess)
+                		expect(guessManager.incorrectGuesses).to.contain(guess)
                 	}
 
                 	it("should not add guess to correctGuesses collection if it is not in the word") {
-                    	let guess = "b"
-                    		guessManager.appendGuess(word, guess: guess)
-                    		expect(guessManager.correctGuesses).notTo.contain(guess)
+                    		let guess = "b"
+                		guessManager.appendGuess(word, guess: guess)
+                		expect(guessManager.correctGuesses).notTo.contain(guess)
                 	}
 	    	}
 	}
```

Comment:
Please take some time to get your IDE set up so that you don't have to worry about or spend so much extra time on things like this.